torch/backends/cudnn/__init__.py

```diff
@@ -9,11 +9,16 @@
 __cudnn_version = None
 # TODO: dynamic version checks via cudnnGetVersion
 
+CUDNN_WIN_LIB = 'cudnn64_6'
```

Comment:
does this hardcode the cuDNN version to 6? That's not a good idea, especially keeping in mind that v7 is out